### DISCUSSION TITLE
Is it possible for Validations to down component hierarchy?

### DISCUSSION THREAD

**Discussion by RealityMasque** [Megabit/Blazorise#5106] [Q&A] [answered]
Hi,

Assume a parent component w/ Validations (let's call it ParentValidations) around all its inputs, & a Submit button that calls ParentValidations.ValidateAll().  Further, assume there is a child component inside the parent component w/ Validations (let's call it ChildValidations) around all its inputs, & this child has a Validate button that calls ChildValidations.ValidateAll().

Is there a way for the parent component's Submit button to cause both its own & the child component's validations to run?

Thanks,

Orion

**Comment by stsrki** [comment]
The short answer is no. We don't support nesting of Validations component. You could probably do it by manually calling `.ValidateAll()` on all the referenced components.

**Reply by RealityMasque** [reply]
by referenced components, do you mean adding "@ref" attributes/properties to ChildValidations?

**Reply by RealityMasque** [reply, marked as answer]
& just learned that providing an @ref to child components in the containing component makes that child component's public methods accessible to the containing component.  yay!
